.github/workflows/ci.yml

```diff
@@ -0,0 +1,109 @@
+name: Continuous Integration
```

Comment:
Minor suggestion: rename it to `CI` to have more space for the rest of the label in the checks summary:
<img width="692" alt="Screenshot 2020-12-05 at 01 49 40" src="https://user-images.githubusercontent.com/766656/101228479-25afac00-369c-11eb-85e2-d462c36de2b6.png">